packages/cli/sw/index.js

```diff
@@ -20,7 +20,10 @@ export function setupRouting() {
 
 	setCatchHandler(({ event }) => {
 		if (isNav(event)) {
-			return caches.match(getCacheKeyForURL('/200.html'));
+			return (
+				caches.match(getCacheKeyForURL('/200.html')) ||
+				caches.match(getCacheKeyForURL('/index.html'))
+			);
 		}
```

Comment:
@prateekbh Sorry, missed that. Hopefully this will work?

Reply:
nvm, need to at least touch up the test suite first

Reply:
Alright, should be corrected now.

Reply:
sry for late reply would `caches.match(getCacheKeyForURL('/200.html') || getCacheKeyForURL('/index.html'))` work?

Reply:
No worries. Should do. Not sure why I did it like that.

Reply:
Corrected. Thanks for the catch.